client/src/components/AppointmentForm.js

```diff
@@ -82,6 +50,30 @@ const AppointmentCreateForm: () => React$Node = () => {
     }
   };
 
+  const fromTimeValue = () => {
+    if (!props.fromTime) {
```

Comment:
instead of using `props.fromTime` and `newFromTime` (and therefore needing to decide which one to use), look up the doc for `useState` and pass `props.fromTime` into it as the initial value.

Reply:
do the same for toTime and address as well.

Reply:
Right. It makes sense because the `variablles` only take value from setState which I just realized.